packages/connect-playwright-example/src/App.tsx

```diff
@@ -14,18 +14,39 @@
 
 import { useCallback, useState, FormEvent, FC } from "react";
 import { ConnectError, createPromiseClient } from "@connectrpc/connect";
-import { createConnectTransport } from "@connectrpc/connect-web";
+import {
+  createGrpcWebTransport,
+  createConnectTransport,
+} from "@connectrpc/connect-web";
 import { ElizaService } from "./gen/connectrpc/eliza/v1/eliza_connect.js";
 
 interface ChatMessage {
   text: string;
   sender: "eliza" | "user";
 }
 
+// Read the transport and format parameters from the URL
```

Comment:
Users won't need to do this. Can you add a comment to explain that we're only doing it for our own test coverage?